### ISSUE TITLE
Link to HTTPClientFactory from the getPage docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4723]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#4723|
|Type|enhancement|
|Created|2010-11-04 13:29:18Z|
|Branch|https://github.com/twisted/twisted/tree/getpage-doc-link-4723|

Most of getPage's arguments are defined by HTTPClientFactory.  The docs already point this out, but do so without generating a link in the resulting API docs.  There should be a link to encourage people to find this information.

<details><summary>Searchable metadata</summary>

```
trac-id__4723 4723
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_getpage_doc_link_4723 branches/getpage-doc-link-4723
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__web web
keywords__documentation documentation
time__1288877358000000 1288877358000000
changetime__1288879252000000 1288879252000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [30210]) Branching to 'getpage-doc-link-4723'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [30211]) Link to HTTPClientFactory

refs [#4723](https://github.com/twisted/twisted/issues/4723)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jonathanj's avatar" src="https://avatars.githubusercontent.com/u/112166?s=50" width="50" height="50">](https://github.com/jonathanj)<a name="note_4"></a>|@jonathanj set owner to @exarkun|
|-|-|

Maybe `@see` should be used or at the very least the word "see" could only be used once in a sentence.

Otherwise it looks good to merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [30213]) Merge getpage-doc-link-4723

Author: exarkun
Reviewer: jonathanj
Fixes: [#4723](https://github.com/twisted/twisted/issues/4723)

Make the `HTTPClientFactory` reference in the `getPage` docstring a link
to help readers find the rest of the relevant documentation.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
